client/index-prod.html

```diff
@@ -8,6 +8,22 @@
         <link rel="icon" type="image/png" href="/dist/stups_favicon.png" />
         <style>${inline}</style>
     </head>
+    <script src="/dist/env.js?v=${timestamp}"></script>
+    <script>
+        var head = document.getElementsByTagName('head')[0],
+                js = document.createElement('script');
+        if (window.APPDYNAMICS_ENABLED) {
+            // load appdynamics
+            js.src = '/dist/adrum.js?v=${timestamp}';
+        } else if (window.YTENV_NEW_RELIC_BROWSER_APP_ID) {
+            // load new relic
+            js.src = '/dist/newrelic-browser.js?v=${timestamp}';
+        }
+        if (js.src && js.src.length != 0) {
+            head.appendChild(js);
```

Comment:
The docs say we should put this right after head, so it would be more like `document.insertBefore(js, body)`?